includes/apple-exporter/class-settings.php

```diff
@@ -30,133 +30,22 @@ class Settings {
 	 * @access private
 	 */
 	private $_settings = array(
-
-		// API information.
-		'api_key' => '',
```

Comment:
All of the formatting settings were moved to the new `Theme` object. What's left in the `Settings` object is everything that isn't configurable through the theme editor.